### PR TITLE
Improve demo content

### PR DESCRIPTION
**Pull request by felipevelzani**
I added some additional examples such as creating a text field and applying the options on a page as an example warning.

### CODE REVIEW

File: source/manifest.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	"name": "Awesome Extension",
 	"version": "0.0.0",
 	"description": "An awesome new browser extension",
-	"homepage_url": "https://github.com/awesome-templates/browser-extension-template",
+	"homepage_url": "https://github.com/fregante/browser-extension-template",
```
Comment (notlmn) [inline source/manifest.json:5]: Nice, thanks!

File: source/manifest.json [hunk 2]
```diff
@@ -15,7 +15,15 @@
 		"128": "icon.png"
 	},
 	"permissions": [
-		"storage"
+		"storage",
+		"<all_urls>"
```
Comment (notlmn) [inline source/manifest.json]: Chrome Web Store (where this extension is automatically published to) would give us a hard time if we do this, requiring justification for every permission we add.

Maybe ask for access to just this repo URL or something along those lines.

File: source/options.html [hunk 3]
```diff
@@ -26,6 +26,14 @@ <h3>Color Previewer</h3>
 		</div>
 		<div class="color-output"></div>
 	</div>
+	<h3>Warning Content</h3>
+	<div class="text-inputs">
+			<label class="text-input">
+				<span>Text</span>
+				<input type="text" name="text">
+			</label>
+		<div class="text-output"></div>
```
Comment (notlmn) [inline source/options.html]: What's happening here, is this element being used? I don't see any changes in `options.js` related to it.

File: source/manifest.json [hunk 4]
```diff
@@ -15,7 +15,15 @@
...
+	],
+	"content_scripts": [ {
+			"matches": [ "*://*/*" ],
+			"js":      [ "content.js" ],
+			"css":     ["content.css"],
+			"run_at":  "document_end"
+		}
```
Comment (notlmn) [inline source/manifest.json]: Weird indentation here for the opening and closing braces, please fix.

**Review by notlmn — CHANGES_REQUESTED** [review]

File: source/options.html (hunk 3 above)
Comment (felipevelzani) [reply source/options.html]: Do you mean the text-output? If so, not using indeed, and kept it by mistake when duplicating the element.

**Comment by felipevelzani** [comment]
Hi @notlmn, thanks a lot for the feedback! 

Feel free to let me know if there is something else that we can improve on this PR. :)

File: source/manifest.json (hunk 4 above)
Comment (fregante) [reply source/manifest.json]: Spacing and indentation hasn't been fixed here

File: source/manifest.json [hunk 5]
```diff
@@ -17,6 +17,14 @@
 	"permissions": [
 		"storage"
 	],
+	"content_scripts": [ 
+		{
+			"matches": [ "https://github.com/fregante/browser-extension-template/*" ],
+			"js":      [ "content.js" ],
+			"css":     [ "content.css" ],
+			"run_at":  "document_end"
```
Comment (fregante) [inline source/manifest.json]: Can you not align the values?

File: source/options.html [hunk 6]
```diff
@@ -26,6 +26,13 @@ <h3>Color Previewer</h3>
 		</div>
 		<div class="color-output"></div>
 	</div>
+	<h3>Warning Content</h3>
```
Comment (fregante) [inline source/options.html]: Can you replace Warning with Notice everywhere?

**Review by fregante — COMMENTED** [review]
Can you change

**Comment by felipevelzani** [comment]
Sure, @fregante! Thank you! Added these changes as well. 😃

**Comment by fregante** [comment]
Thanks! I just noticed I sent some extra "can you change" text 😂

**Comment by felipevelzani** [comment]
I see that it is still showing with the changes requested status. Feel free to let me know if there is something that I can do to make it happen. :)

**Review by notlmn — APPROVED** [review]

**Pull request merged**